mautic/mautic
# Respect timezone on best read hours widget

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:
The best read hours widget display data in UTC timezone. This PR add support for timezone from configuration


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Set Europe/Paris timezone in configuration
3. Send email and open one of them
4. Add Best read hours widget on dasboard
5. Before PR you see hours range in UTC, after PR you see hours range in Euro/Paris timezone
